packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts

```diff
@@ -573,7 +573,7 @@ export class LocalTunnelTaskTerminal extends BaseTaskTerminal {
       ngrokArgs.push("--log-format=logfmt");
     }
 
-    return `${ngrokPath} ${ngrokArgs.join(" ")}`;
+    return `"${ngrokPath}" ${ngrokArgs.join(" ")}`;
```

Comment:
Does `""` work on Mac and Linux as well?

Reply:
![image](https://user-images.githubusercontent.com/49138419/210479635-b11632d6-c263-425c-8c36-079e5dd2bacf.png)
I tested it in WSL.